### ISSUE TITLE
Arrays contain "None" (string) instead of None (object)

### ISSUE THREAD

**Issue by Nikratio** [astrofrog/idlsave#3]
If there is no data in some array elements, IDLsave returns the string "None" for these elements, rather than the None object.

An example sav file is available on http://www.rath.org/res/sensor_map.sav, look at the contents of the variable "TAr_names".

**Comment by astrofrog** [comment]
Thanks for the report - I think this was fixed in the git repository since the last stable release. I get:

```
In [3]: s.tar_names
Out[3]: 
array([[, , , , , , , , , ],
       [TA01_S2R, TA02_S2R, TA03_S2R, TA04_S2R, TA05_S2R, TA06_S2R,
        TA07_S2R, TA08_S2R, TA09_S2R, TA10_S2R],
       [, , , , , , , , , ]], dtype=object)
```

(i.e. it now return an empty string which is the way it should behave, rather than returning `None` or `'None'`)

Let me know if this issue is already resolved in the latest git version.

**Comment by astrofrog** [comment]
I am closing this ticket since the issue should be fixed in c39f3ae726b4e8c82e87 - please re-open the ticket if not.
